models/recurly__subscription_overview.sql

```diff
@@ -7,16 +7,15 @@ with subscription_history as (
 
 plan_history as (
 
-    select * 
-    from {{ var('plan_history') }}
+    select *  
+    from {{ var('plan_history') }} 
 ),
 
 subscription_enhanced as (
 
     select 
         *,
-        coalesce(canceled_at, current_period_ended_at) as subscription_end_date,
-        row_number() over (partition by subscription_id order by current_period_started_at) - 1 as subscription_period
```

Comment:
This is already generated in the source model and causes an ambiguous error if we leave it here. We should choose to either keep the logic in this model, or the staging.

Reply:
Yep, accidentally merged it back into the source model yesterday. Removed it from source for now, figured it'd be best to make any new non-trivial field creation occur in the transform side.